emaple.py

```diff
@@ -7,7 +7,7 @@ def meles():
 
     def paul():
         for i in range (10):
-            print("done")
+            print("stop")
```

Comment:
paul, if you can change the print from stop to checking loop function, that would be awesome, Thank you

Reply:
I resolved it Meles.